src/components/DownloadContactListButton.tsx

```diff
@@ -8,22 +8,29 @@ type DownloadButtonProps = {
 }
 
 export function DownloadContactListButton (props: DownloadButtonProps) {
-    const csvHeader = '"Property Address","Property Type (Marketing)","Owner","Owner Mobile","Owner Phone","Contact","Contact Mobile","Contact Work Phone"\r\n'
+    const csvHeader = '"Property Address","Name","Owner/Contact","Mobile","Work Phone"\r\n'
     const arrayOfPropertyObjects = props.results
 
-    const uniqueProperties = getUniqueListBy(arrayOfPropertyObjects, 'Deal_Name')
+    const uniqueProperties = getUniqueListBy(arrayOfPropertyObjects, 'id')
+    let type = 'unknown'
     const csvRows = uniqueProperties.map((result: UnprocessedResultsFromCRM) => {
         if (result.owner_details && Array.isArray(result.owner_details)) {
             const mobileNumbers = result.owner_details.map((owner: OwnerType) => owner.Mobile).filter((Mobile: string) => Mobile)
             const mobile = mobileNumbers.length > 0 ? mobileNumbers[0] : null
             const workPhones = result.owner_details.map((owner: OwnerType) => owner.Work_Phone).filter((Work_Phone: string) => Work_Phone)
             const workPhone = workPhones.length > 0 ? workPhones[0] : null
             const propertyAddress = result.Deal_Name
-            const propertyTypeMarketing = result.Property_Category_Mailing
-            const ownerData = result.owner_details.find((owner: OwnerType) => owner.Contact_Type === 'Owner')
-            const contactData = result.owner_details.find((owner: OwnerType) => owner.Contact_Type === 'Director')
+            const ownerDetails = result.owner_details
+            const contact: OwnerType = ownerDetails[0]
```

Comment:
I don't think this logic is correct. Why was it changed from ```const ownerData = result.owner_details.find((owner: OwnerType) => owner.Contact_Type === 'Owner')```

Reply:
I'm not sure. @haydenf why the change?

Reply:
Because not every related contact is of contact type director, where related contact is seems to always be ownerdetails[0] and owner is ownerdetails[1]

Reply:
yeah that's why we should use const ownerData = result.owner_details.find((owner: OwnerType) => owner.Contact_Type === 'Owner')

We cannot assume that index 0 is always the related contact

Reply:
This is still an issue